lib/dynosaur/process/heroku.rb

```diff
@@ -0,0 +1,42 @@
+require 'dynosaur/client/heroku_client'
+
+require_relative '../process'
+
+# Start a detached rake task on a one-off dyno
+module Dynosaur
+  class Process
+    class Heroku < Process
+      # Valid dyno sizes under Heroku's new pricing model
+      module Size
+        PERFORMANCE_M = 'performance-m'.freeze
+        PERFORMANCE_L = 'performance-l'.freeze
+        STANDARD_1X = 'standard-1X'.freeze
+        STANDARD_2X = 'standard-2X'.freeze
```

Comment:
Generally, alphabetical is a good default, but since this is a small set that doesn't change, in order of size would be helpful to readers.